pkgs/by-name/te/tenv/package.nix

```
@@ -0,0 +1,43 @@
+{ buildGoModule, fetchFromGitHub, lib, testers, tenv }:
+
+buildGoModule rec {
+  pname = "tenv";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "tofuutils";
+    repo = "tenv";
+    rev = "v${version}";
+    hash = "sha256-yLNdBwKF6Jts661P+YZhFGNr71TG7Scb6RGvFxTLqYQ=";
+  };
+
+  vendorHash = "sha256-GGWiP1rIDF6qxST2ZmnKNkgbS+15hxaCs1d1+UEiYgU=";
+
+  # Tests disabled for requiring network access to release.hashicorp.com
+  doCheck = false;
+
+  ldflags = [
+    "-s" "-w"
+    "-X main.version=v${version}"
+  ];
+
+  postInstall = ''
+    installShellCompletion --cmd tenv \
```

```suggestion
    installShellCompletion --cmd tenv \
```
This package was not declared:
> installShellCompletion: command not found

---

Solution:
>  nativeBuildInputs = [ installShellFiles ];

(Add to build phase.)

---

That error also shows you haven't built before pushing the code.
See how important is the habit? I suggest you learn the habit for the sake of learning the habit. It saves everyone time...

---

True, I might add a git pre-hook on the repo just to avoid this :)